app/modules/conversations/session/session_service.py

```
@@ -0,0 +1,163 @@
+            prefix = f"chat:stream:{conversation_id}:"
+            run_id = key_str[len(prefix) :]
+
+            # Check if stream still exists and has activity
+            if not self.redis_manager.redis_client.exists(active_key):
+                return ActiveSessionErrorResponse(
+                    error="No active session found", conversationId=conversation_id
+                )
+
+            # Get stream info to determine cursor position
+            try:
+                stream_info = self.redis_manager.redis_client.xinfo_stream(active_key)
```

Ruff (0.13.1)

Local variable `stream_info` is assigned to but never used

Remove assignment to unused variable `stream_info`

(F841)